pennylane/ops/op_math/decompositions/two_qubit_unitary.py

```diff
@@ -291,8 +293,8 @@ def _decomposition_0_cnots(U, wires):
      -╰U- = -B-
     """
     A, B = _su2su2_to_tensor_products(U)
-    A_ops = one_qubit_decomposition(A, wires[0])
-    B_ops = one_qubit_decomposition(B, wires[1])
+    A_ops = one_qubit_decomposition(A, wires[0], return_global_phase=True)
+    B_ops = one_qubit_decomposition(B, wires[1], return_global_phase=True)
```

Comment:
I thought `A` and `B` are expected to be SU(2) here?

Reply:
I think it could be the case that even if A and B has not global phase, the decomposition adds it.
If I remove those returns, I get an error in [this test. ](https://github.com/PennyLaneAI/pennylane/blob/893ba9fce366a2eb1d39dc4b212809285da87ee0/tests/ops/qubit/test_matrix_ops.py#L499)

However, following your next comment, I removed the returns and It works correctly